internal/pkg/keymanager/error.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2021 Apptainer a Series of LF Projects LLC
+//   For website terms of use, trademark policy, privacy policy and other
+//   project policies see https://lfprojects.org/policies
+// Copyright (c) 2020, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the LICENSE.md file
+// distributed with the sources of this project regarding your rights to use or distribute this
+// software.
+
+package keymanager
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	jsonresp "github.com/sylabs/json-resp"
```

Comment:
Did you intend to still rely on a sylabs module?

Reply:
@DrDaveD yes - that needs to be a separate PR -- been doing one PR per module reference

Reply:
Ok